cli/rescript.js

```diff
@@ -18,7 +18,8 @@ try {
       subcommand === "build" ||
       subcommand === "watch" ||
       subcommand === "clean" ||
-      subcommand === "compiler-args"
+      subcommand === "compiler-args" ||
```

Comment:
Nitpick: Maybe we can collect all subcommands in a `Set` and call .contains?

Reply:
Or use a `switch` statement.